### DISCUSSION TITLE
Stopping the screening on ASReview

### DISCUSSION THREAD

**Discussion by cchiara1** [asreview/asreview#1655] [Q&A] [answered]
Dear ASReview community,

I have a basic question to confirm that I understand the following instructions right: 

"After deciding to stop screening, you can [export the results](https://asreview.readthedocs.io/en/latest/progress.html#export-results) (i.e., the partly labeled data and the [project file](https://asreview.readthedocs.io/en/latest/example_api_asreview_file.html) containing the technical information to reproduce the entire process) and publish them on, for example, the Open Science Framework. As a final step, you can mark the project as [finished](https://asreview.readthedocs.io/en/latest/progress.html#mark-project-as-finished) in ASReview."

Based on this extract, if my stopping criterion of 100 irrelevant papers has been reached, then I would consider all the papers marked as relevant until this point to be a comprehensive list of relevant papers (with some margin of error) and proceed to the screening of their full text?

Thank you in advance.

**Comment by FelixWdm** [comment, marked as answer]
Dear @cchiara1, that is correct. There is indeed some margin of error, there could be relevant papers in the unseen set. But additional search methods can retrieve these (and additional papers) more efficiently. See for instance: 

van Haastrecht, M., Sarhan, I., Yigit Ozkan, B., Brinkhuis, M., & Spruit, M. (2021). SYMBALS: A systematic review methodology blending active learning and snowballing. Frontiers in research metrics and analytics, 6, 685591. https://doi.org/10.3389/frma.2021.685591

**Reply by cchiara1** [reply]
Thank you for the useful answer!
